jujugui/static/gui/src/app/components/expert-card/expert-card.js

```diff
@@ -0,0 +1,141 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const GenericButton = require('../generic-button/generic-button');
+const SvgIcon = require('../svg-icon/svg-icon');
+
+const experts = {
+  spicule: {
+    supportedEntities: ['drillbit'],
```

Comment:
We aren't pre-defining this list. If the owner matches one of these top level keys and the charm has metrics and plans then it'll get the card.